cli/cmd/cluster.go

```diff
@@ -176,25 +204,21 @@ func newGatewaysCommand() *cobra.Command {
 	return cmd
 }
 
-func newSetupRemoteCommand() *cobra.Command {
-	options, err := newSetupRemoteClusterOptionsWithDefault()
+func newMulticlusterInstallCommand() *cobra.Command {
+	options, err := newMulticlusterInstallOptionsWithDefault()
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%s", err)
 		os.Exit(1)
 	}
 
 	cmd := &cobra.Command{
 		Hidden: false,
-		Use:    "setup-remote",
-		Short:  "Sets up the remote cluster by creating the gateway and necessary credentials",
+		Use:    "install",
+		Short:  "Output Kubernetes configs to install the Linkerd multi-cluster add-on",
```

Comment:
Should we have some long-form description here listing the components (at a conceptual level) that this will install?  e.g. the service-mirror, the gateway, and the service account?  Or is this information better suited for a documentation page?